src/static/styles/App.scss

```diff
@@ -151,6 +172,25 @@ body {
 	border-color: $primary-theme;
 }
 
+
+.btn-outline-primary-continue {
+	color: $primary-theme;
+	background-color: white;
+	border-color: $primary-theme;
+	width: 120px;
+}
+
+.btn-outline-primary-continue:hover,
+.btn-outline-primary-continue:focus,
+.btn-outline-primary-continue:active,
+.btn-outline-primary-continue.active,
+.open > .dropdown-toggle.btn-outline-primary-continue {
+	color: white;
+	background-color: $primary-theme;
+	border-color: $primary-theme;
+	width: 120px;
```

Comment:
try to move away from px and to use em. This shouldn't have passed the husky runner. Are you running with --no-verify? You shouldn't